src/firewall/varktables/types.rs

```diff
@@ -234,7 +237,21 @@ pub fn get_network_chains<'a>(
     chains.push(postrouting_chain);
 
     // FORWARD chain
-    let mut forward_chain = VarkChain::new(conn, FILTER.to_string(), FORWARD.to_string(), None);
+    let mut forward_chain: VarkChain<'_> = VarkChain::new(conn, FILTER.to_string(), FORWARD.to_string(), None);
+    
+    // INPUT chain
+    let mut input_chain: VarkChain<'_> = VarkChain::new(conn, FILTER.to_string(), INPUT.to_string(), None);
+
+    // Always add ACCEPT rules in firewall for dns traffic from containers
+    // to gateway when using bridge network with internal dns.
+    input_chain.build_rule(VarkRule::new(
+        format!(
+            "-p {} -s {} --dport {} -j {}",
+            "udp", network, dns_port, ACCEPT
+        ),
+        Some(TeardownPolicy::OnComplete),
+    ));
+    chains.push(input_chain);
```

Comment:
we should create another chain in the INPUT chain called NETAVARK_INPUT, then put this rule into the NETVARK_INPUT chain. This makes sure to keep our rules better organised.

see the NETAVARK_FORWARD chain as an example